simulator/src/main/java/com/hedera/block/simulator/BlockStreamSimulatorApp.java

```diff
@@ -30,14 +36,14 @@ public class BlockStreamSimulatorApp {
     private static final System.Logger LOGGER =
             System.getLogger(BlockStreamSimulatorApp.class.getName());
 
-    Configuration configuration;
-    BlockStreamManager blockStreamManager;
-    PublishStreamGrpcClient publishStreamGrpcClient;
-    BlockStreamConfig blockStreamConfig;
+    private final Configuration configuration;
+    private BlockStreamManager blockStreamManager;
+    private PublishStreamGrpcClient publishStreamGrpcClient;
+    private BlockStreamConfig blockStreamConfig;
 
     private final int delayBetweenBlockItems;
 
-    boolean isRunning = false;
+    private boolean isRunning = false;
```

Comment:
Just a comment for the future, since this has no usage now.
I have a very strong feeling that this field would be used to stop the app, which would very possibly be done from another thread. If that would be the case, this field should become volatile in order to have the visibility (flush the data beyond CPU cache in order for other threads to see it) and the so called 'happens before guarantees'. Without that, we can expect that this flag will not function properly. Also, if we are going to change the data of the flag in a non atomic fashion, then we would need to lock.